components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -43,7 +43,7 @@ const (
 	PublisherSecretEMSURLKey        = "ems-publish-url"
 	PublisherSecretBEBNamespaceKey  = "beb-namespace"
 
-	configMapName               = "eventing"
+	configMapName               = "eventing-configmap"
```

Comment:
I really had to do this change because I was not able to create a configmap with `eventing`